### ISSUE TITLE
Last frame is centered on end?

### ISSUE THREAD

**Issue by turian** [marl/crepe#73]
The README says that the default is center-frames, so the first frame is centered as 0 and only a half-size frame.

Is the same true of the last frame? If so can the README be updated?

**Comment by sharvil** [comment]
Yes, an empty half frame is [appended to the end as well](https://github.com/marl/crepe/blob/master/crepe/core.py#L198).
